sdk/security/keyvault/azadmin/backup/utils_test.go

```diff
@@ -55,25 +53,22 @@ func run(m *testing.M) int {
 		}()
 	}
 
-	if recording.GetRecordMode() == recording.PlaybackMode {
-		credential = &FakeCredential{}
-	} else {
-		tenantID := lookupEnvVar("AZADMIN_TENANT_ID")
-		clientID := lookupEnvVar("AZADMIN_CLIENT_ID")
-		secret := lookupEnvVar("AZADMIN_CLIENT_SECRET")
-		var err error
-		credential, err = azidentity.NewClientSecretCredential(tenantID, clientID, secret, nil)
-		if err != nil {
-			panic(err)
-		}
+	var err error
+	credential, err = azcred.New(nil)
+	if err != nil {
+		panic(err)
 	}
 
-	hsmURL = getEnvVar("AZURE_MANAGEDHSM_URL", fakeHsmURL)
-	blobURL = getEnvVar("BLOB_CONTAINER_URL", fakeBlobURL)
-	token = getEnvVar("BLOB_STORAGE_SAS_TOKEN", fakeToken)
+	hsmURL = recording.GetEnvVariable("AZURE_MANAGEDHSM_URL", fakeHsmURL)
+	blobURL = recording.GetEnvVariable("BLOB_CONTAINER_URL", fakeBlobURL)
+	token = recording.GetEnvVariable("BLOB_STORAGE_SAS_TOKEN", fakeToken)
```

Comment:
No need to define a local constant having another constant's value; just reference that other constant
```suggestion
	token = recording.GetEnvVariable("BLOB_STORAGE_SAS_TOKEN", recording.SanitizedValue)
```

Reply:
fixed in 2b6cf72